app/controllers/comments_controller.rb

```diff
@@ -1,13 +1,24 @@
 class CommentsController < ApplicationController
   load_and_authorize_resource
   before_action :set_user_and_post
```

Comment:
- [ ] It is still difficult to create a _**comment**_ for a `Post` instance due to **_the CRSF token_** issue. 

![image](https://github.com/ismayito/Rails-blog-app/assets/77106962/d9ec5e88-a7b9-4ff0-b42e-3fd76a525cd9)

---

For you to successfully start, you can add this piece of code at the top of the controller 👍 💻 🖱️:

```ruby
  skip_before_action :verify_authenticity_token, only: %i[create]
```